CHANGES.rst

```diff
@@ -82,8 +82,8 @@ Release History
   when solving for weights.
   (`#1507 <https://github.com/nengo/nengo/pull/1507>`__)
 - Solvers no longer take encoders as an argument. Instead, encoders will
-  be applied to the targets before the solve function for compositional solvers
-  and applied by the Transform builder for non-compositional solvers.
+  be applied to the targets before the solve function for non-compositional
+  solvers and applied by the Transform builder for compositional solvers.
```

Comment:
Note: This still only applies to the case of `weights=True`. For the case of `weights=False` there should not be a difference. This is now clarified in the `Solver` documentation.